.github/workflows/abi.yaml

```diff
@@ -110,7 +122,7 @@ jobs:
           cp build_abi/install_lib/* `pg_config --pkglibdir`
           chown -R postgres /mnt
           set -o pipefail
-          sudo -u postgres make -C build_abi -k regresscheck regresscheck-t regresscheck-shared IGNORES="memoize" | tee installcheck.log
+          sudo -u postgres make -C build_abi -k regresscheck regresscheck-t regresscheck-shared IGNORES="memoize ${{matrix.ignores}}" | tee installcheck.log
```

Comment:
Would be better if you move everything to new `matrix.ignores` variable for all configurations instead of doing part of it, no?? I mean it just for better readability